mixnode/src/node/packet_delayforwarder.rs

```diff
@@ -61,10 +61,11 @@ where
         let packet_mode = packet.packet_mode();
         let sphinx_packet = packet.into_sphinx_packet();
 
-        if let Err(err) =
-            self.mixnet_client
-                .send_without_response(next_hop, sphinx_packet, packet_mode)
-        {
+        if let Err(err) = self.mixnet_client.send_without_response(
+            next_hop,
+            NymPacket::Sphinx(sphinx_packet),
```

Comment:
I might be missing something, or it will come in next PR, but what if it's an outfox packet?